### ISSUE TITLE
Add clarification on copyright of generated messages files

### ISSUE THREAD

**Issue by iluetkeb** [ros/gencpp#27]
In #20, the copyright statement was removed from generated header files. However, this creates an ambiguity. We don't know whether the generated source code has the copyright of the msg file, or the copyright of gencpp (because gencpp creates it).

It would be helpful if the gencpp license statement could be extended to explicitly state what copyright the generated messages are under. Preferably, that should be the copyright of the input file, but BSD would also be acceptable.

Sorry if this seems overly picky, but you know, any ambiguity is bad in legal questions.

**Comment by dirk-thomas** [comment]
The generated code does not have the same copyright as `gencpp`. Therefore it has been removed.

I don't know what kind of statement you think should be added. Please consider providing a pull request for it.

**Comment by iluetkeb** [comment]
Well, I'm not a lawyer, but I think the suggested clarification appropriately conveys the intention.

**However**, there are two problems with this:

Firstly, and most importantly,
- I don't really know who the Copyright holders are. I've specified everybody who was in the "git log" output, then aggregated everybody @willowgarage.com and everybody @osrfoundation.org to their respective organizations, and listed the remainder individually.
- I can't know whether the contributions by people using their private e-mail are actually owned by their employers, we would have to ask, or maybe you know.
- I also made a judgment call that commits entries containing to Dirk's private e-mail address are actually work for the OSRF.
- I made another judgment call to ignore Scott K Logan's one-line addition to the package.xml

Secondly, more or less as a corollary, we can't make the amendment specifying that generated source code has the copyright and license of the input without asking the copyright holders.
